includes/admin/class-wc-payments-admin-additional-methods-setup.php

```diff
@@ -26,6 +37,15 @@ public function enqueue_scripts() {
 			return;
 		}
 
+		$available_methods = $this->gateway->get_upe_available_payment_methods();
+		if ( empty( $available_methods ) ) {
+			return;
+		}
+
+		if ( count( $available_methods ) <= 1 ) {
```

Comment:
```suggestion
		if ( 1 >= count( $available_methods ) ) {
```
Just some Yoda condition